libs/util/src/lib/extractIds.ts

```diff
@@ -0,0 +1,44 @@
+export enum extractIdType {
+  tournament = 'tournament',
```

Comment:
Please use `PascalCase` for enum identifiers.

Reply:
Changed in #14 under commit ef2a597785f6a6f2133243b6e7423cd2d58adbde